configs/common/Options.py

```diff
@@ -280,12 +282,12 @@ def addCommonOptions(parser, configure_xiangshan=False):
                         action=ListRP, nargs=0,
                         help="List available replacement policy types")
 
+    parser.add_argument("--kmh-align", action="store_true", default=False,
+                        help="""
+                        Use kmu config""")
     parser.add_argument("--list-hwp-types",
                         action=ListHWP, nargs=0,
                         help="List available hardware prefetcher types")
-    parser.add_argument("--l1d-use-xsstride", action="store_true", default=False,
-                        help="""
-                        Enable SPP component for L1 data prefetcher""")
```

Comment:
@jueshiwenli Is it ok to delete this option?

Reply:
Now use kmh-align argument to switch this

Reply:
I think it is ok, kmh-align can provide the same function.